README.md

```diff
@@ -55,8 +69,222 @@ documentation for more details.
   for details.
 * The workflow expects the GitHub repository to include secrets with AWS
   credentials (`AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`).
+  
+## AWS Credentials
+
+There are two ways to supply AWS credentials to the workflow. The default is to
+store long-lived AWS user credentials in GitHub secrets. Alternatively, you can
+use an IAM role that trusts the GitHub OpenId Connect provider to authenticate.
+Using OIDC to authenticate is not the default because it is a newer option, but
+there are compelling benefits to this approach. You can read more
+[here](https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/about-security-hardening-with-openid-connect).
+
+### GitHub Secrets
+
+Authenticating via this approach means that you will be manually creating AWS
+credentials and duplicating them in GitHub secrets. The workflow expects the
+GitHub repository to include secrets with AWS credentials under 
+`AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`. You can override these defaults 
+by supplying the `awsCredentials` property to the workflow:
+
+```ts
+import { App } from 'aws-cdk-lib';
+import { ShellStep } from 'aws-cdk-lib/pipelines';
+import { GithubWorkflow } from 'cdk-pipelines-github';
+
+const app = new App();
+
+const pipeline = new GithubWorkflow(app, 'Pipeline', {
+  synth: new ShellStep('Build', {
+    commands: [
+      'yarn install',
+      'yarn build',
+    ],
+  }),
+  awsCredentials: {
+    accessKeyId: 'MY_ID',
+    secretAccessKey: 'MY_KEY',
+    sessionToken: 'MY_TOKEN', // default is no token
```

Comment:
Tokens don't make sense here, as long-lived credentials never have tokens.

Leave the parameter in, because who knows what an enterprising user might use these for. But there's not really a point to wasting valuable brain space for doc readers on including this field.